### DISCUSSION TITLE
model fiji bestway

### DISCUSSION THREAD

**Discussion by Remigiusz2020** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#546] [General]
Hi,
this project it is a great idea. Can someone help me to check , how many wires my jacuzzi have. How can I get to the inside? 
 I have square model with pump S100101 - like model Santa lucia/rio. It is confirmed that is working with 1+2 . Is it mean 1 or 2 ?

**Comment by visualapproach** [comment]
Hi, you have to remove cover by unscrewing some screws on the pump's "waist". 1+2 means either is reported to work if you're talking about LLC in the manual.

**Reply by Remigiusz2020** [reply]
thank you very much. Do you know who can have any spare v2 board?

**Comment by foersst** [comment]
I also have the model Fiji with pump S100101. Can somebody confirm that this combination is compatible?
